doc/stolas_privacy_policy_2.md

```diff
@@ -0,0 +1,75 @@
+# Privacy Policy
+
+**Entity:** Lemon Tree Technologies LTD (BVI) 
+**Project:** stOLAS  
+**Website:** [https://lstolas.xyz](https://lstolas.xyz)  
+**Contact:** info@lstolas.xyz  
+
+---
+
+## 1. Introduction
+Lemon Tree Technologies LTD (“we,” “our,” or “us”) respects your privacy and is committed to protecting your personal information.  
+This Privacy Policy explains how we handle information in connection with your use of the stOLAS website and decentralized application (collectively, the “Platform”).
+
+---
+
+## 2. Information We Do Not Collect
+The Platform is **non-custodial** and operates without user registration.  
+We do **not** collect or store personal data such as names, email addresses, identification numbers, or financial information.  
+We also do not have access to your private keys or wallets — all transactions occur directly on the blockchain.
+
+---
+
+## 3. Information Automatically Collected by Hosting Provider
+Our Website is hosted on **Vercel**, which may automatically and temporarily process limited technical data for operational, security, and performance purposes.  
```

Comment:
Let's use ask the frontend people to render this markdown file. 
So we don't need to update anything and can just render the blocks directly from gh